Cargo.lock

```diff
@@ -2,6 +2,21 @@
 # It is not intended for manual editing.
 version = 3
 
+[[package]]
```

Comment:
Remove the Cargo.lock, remove the target directory do cargo build and commit with the updated Cargo.lock not this one